### PR TITLE
Improves the Clockwork Slab's Recollection

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: The Recollection option in the Clockwork Slab has been significantly improved, with a better information to fluff ratio, and the ability to toggle which tiers of scripture that are visible in Recollection.
/:cl:

This does increase the amount of lines you're expected to read, but there's something useful in all of them instead of most of them being unimportant fluff.

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_items/clockwork_slab.dm [hunk 1]
```diff
@@ -250,43 +253,77 @@
...
+			Each tier has additional requirements, including Servants, Tinkerer's Caches, and <b>Construction Value</b>(<b>CV</b>). Construction Value is gained by creating structures or converting the \
+			station, and everything too large to hold will grant some amount of it.<br><br>\
+			\
+			This would be a massive amount of information to try and keep track of, but all Servants have the <b><font color=#BE8700>Global Records</font></b> alert, which appears in the top right.<br>\
+			Mousing over that alert will display Servants, Caches, CV, and other information, such as the tiers of scripture that are unlocked.<br><br>\
+			\
+			On that note, <font color=#BE8700>Scripture</font> is recited through <b><font color=#BE8700>Recital</font></b>, the first and most important function of the slab.<br>\
+			All scripture requires some amount of <font color=#BE8700>Components</font> to recite, and only the weakest scripture does not consume any components when recited.<br>\
+			However, weak is relative when it comes to scripture; even the 'weakest' could be enough to dominate a station in the hands of cunning enough Servants, and higher tiers of scripture are \
```
Comment (ShadowDeath6) [inline code/game/gamemodes/clock_cult/clock_items/clockwork_slab.dm]: "cunning enough servants" sounds weird. Might be better as "enough cunning servants" or "cunning servants"

**Comment by Xhuis** [comment]
A big improvement, but I still think it's too wordy. A lot of its information can be conveyed with simpler words and doing so would help contribute to readability.
